source/_lovelace/entity-filter.markdown

```diff
@@ -19,20 +19,27 @@ This type of card can also be used together with rest of cards that allow multip
 Screenshot of the entity filter card.
 </p>
 
-## {% linkable_title Options %}
-
 {% configuration %}
 type:
   required: true
   description: entity-filter
   type: string
 entities:
   required: true
-  description: Array of entity_ids
-  type: array
+  description: "List of entities to show."
+  type: list
+  keys:
+    name:
+      required: false
+      description: The name to use in the front for the entity.
+      type: string
+    entity:
+      required: false
+      description: "The `entity_id` to show in the frontend."
+      type: string
 state_filter:
   required: true
-  description: Array of strings representing states
+  description: Array of strings representing states.
```

Comment:
we should only use one word `array` or `list`. afail `list` is more common in the docs

Reply:
Agree